Makefile

```diff
@@ -19,6 +19,10 @@ NVCUFLAGS += -DUSE_MPI_PARTITIONED
 SRCFILES += src/partitioned.cu
 endif
 
+ifeq ($(MPI_ACX_MEMOPS_V2), 1)
+NVCUFLAGS += -USE_MEMOPS_V2
```

Comment:
I just noticed this, too. 🤦  Should this be `-DUSE_MEMOPS_V2`?

Reply:
Yes (amended)